scripts/controlnet.py

```diff
@@ -35,10 +36,10 @@
 CN_MODEL_EXTS = [".pt", ".pth", ".ckpt", ".safetensors"]
 cn_models = {}      # "My_Lora(abcd1234)" -> C:/path/to/model.safetensors
 cn_models_names = {}  # "my_lora" -> "My_Lora(abcd1234)"
-cn_models_dir = os.path.join(scripts.basedir(), "models")
+cn_models_dir = os.path.join(models_path, "ControlNet")
 os.makedirs(cn_models_dir, exist_ok=True)
+default_conf = os.path.join(scripts.basedir(), "models", "cldm_v15.yaml")
```

Comment:
For backward compatibility, we should also consider the original model path in case some one need it

Reply:
Okay, I added the old path as well.